test/draw/RasterMonochromeTest.ooc

```diff
@@ -45,6 +45,14 @@ RasterMonochromeTest: class extends Fixture {
 			expect(image1 equals(image2))
 			image1 referenceCount decrease(); image2 referenceCount decrease()
 		})
+		this add("lossy ascii serialization", func {
+			image1 := RasterMonochrome open(this sourceSpace)
+			alphabet := " .,-_':;!+~=^?*abcdefghijklmnopqrstuvwxyz()[]{}|&%@#0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+			asciiImage := image1 toAscii(alphabet)
+			image2 := RasterMonochrome fromAscii(asciiImage)
+			expect(image1 distance(image2) < 0.01)
```

Comment:
Sorry, just one more thing.
`expect(image1 distance(image2), is lessThan(0.01f))` (note that `distance` returns a `Float` as well)